src/ui/component/common/hidden-nsfw.jsx

```diff
@@ -0,0 +1,21 @@
+// @flow
+import React from 'react';
+import Button from 'component/button';
+import * as ICONS from 'constants/icons';
+import Icon from 'component/common/icon';
+
+type Props = {
+  type?: string,
+};
+
+export default function HiddenNsfw(props: Props) {
+  const { type = 'page' } = props;
+
+  return (
+    <div className="section--padded section__subtitle">
+      <Icon icon={ICONS.EYE_OFF} />
+      {__(' Content might be hidden on this ' + type + ' because of your ')}
```

Comment:
Please change this to:

```
<I18nMessage tokens={{ type, settings: <Button button="link" label={__('settings')} href="/$/settings" /> }}>
  Content may be hidden on this %type% because of your %settings%
</I18nMessage>
```

Reply:
I updated this 👍